parsers/US_CA.py

```diff
@@ -56,6 +56,15 @@ def get_target_url(target_datetime: Optional[datetime], kind: str) -> str:
     return target_url
 
 
+def add_production_to_dict(mode: str, value: float, production_dict: dict) -> dict:
+    """Add production to production_dict, if mode is in PRODUCTION_MODES."""
+    if PRODUCTION_MODES_MAPPING[mode] not in production_dict:
+        production_dict[PRODUCTION_MODES_MAPPING[mode]] = value
+    else:
+        production_dict[PRODUCTION_MODES_MAPPING[mode]] += value
+    return production_dict
```

Comment:
I think we should consider making this a lib function as it would be usefull in other parsers as well.

Reply:
noted, i think i've already seen this logic in a parser as well

Reply:
will merge for now though so that i can trigger the data refetch